applications/DG-Max/ElementInfos.cpp

```diff
@@ -159,6 +200,26 @@ double jelmerStructure(const Geometry::PointPhysical<DIM>& pPhys,
         // std::cout << pPhys[0] << " " << pPhys[1] << " " << pPhys[2]  << "
         // " << epsilon_ << "\n";
 
+    } else if (structureType == 5) {
```

Comment:
can we at least make the structure Type an enum?

Reply:
Should have known this comment was coming, I'll see what I can do. Problem is that the input goes via a command line argument.

Reply:
Done